nomenclate/core/configurator.py

```diff
@@ -13,53 +15,47 @@
 import os
 from collections import OrderedDict
 import nomenclate.core.toolbox as tb
-
+import nomenclate.core.exceptions as exceptions
 
 
 class ConfigParse(object):
-    def __init__(self, path=None):
+    def __init__(self, config_filepath=None):
         """
         Args:
             section (str): section to query
             subsection (str):subsection to query
         """
-        if not path:
-            path = tb.get_config_filepath()
-        self.path = path
-        self.data = None
+        if not config_filepath:
+            config_filepath = tb.get_config_filepath()
+        self.path = config_filepath
```

Comment:
Yeah, this was one of the workaround for the Maya  style coding environment.  I realized toolbox isn't incredibly useful at this point so I will phase that out to a relative path for a config.